### ISSUE TITLE
Firefox autocomplete with more than 1 account not working

### ISSUE THREAD

**Issue by Jump3r99** [flarum/core#698]
I saved passwords for 2 accounts in FF. When I'm trying to select one in logging window FF autocomplete only password - I have to select once more same nick.

**Comment by Luceos** [comment]
This sounds like a browser issue, can you explain a bit more how this is a Flarum issue?

**Comment by Jump3r99** [comment]
I think flarum wrong switch between login data. Password seem to be correct.

**Comment by dcsjapan** [comment]
Confirmed with Firefox 43.0 for OS X. I created a test account on discuss.flarum.org to test this. When I type a "D" in the Username field, Firefox presents a dropdown list allowing me to select "Dominion" as the user name. It then fills in the password for that account, but *does not* fill in the username. I have to type "D" and select "Dominion" from the dropdown list a second time to get the Username field filled.

In other words, it's just as @Jump3r99 described.

I was tempted to think it's a browser issue, but I tested the same thing with two accounts on our esoTalk forums. There, the Firefox autofill function enters *both* the username *and* password* correctly.

It seems possible that Flarum is doing something that's preventing autofilling of the Username field.

**Comment by franzliedke** [comment]
This may be related to #467.

**Comment by tobscure** [comment]
Fixed in #811
